setup.cfg

```
@@ -1,3 +1,9 @@
 # Inside of setup.cfg
 [metadata]
 description-file = README.md
+
+[isort]
+# https://pycqa.github.io/isort/docs/configuration/options.html
+line_length = 120
```

```suggestion
line_length = 100
```
we have set 100 in the black config